addons/l10n_eg/views/account_tax.xml

```diff
@@ -0,0 +1,23 @@
+<data>
+        <record id="view_account_tax_form" model="ir.ui.view">
+            <field name="name">account.tax.form</field>
+            <field name="model">account.tax</field>
+            <field name="inherit_id" ref="account.view_tax_form"/>
+            <field name="arch" type="xml">
+                <field name="name" position="after">
+                    <field name="l10n_eg_eta_code" attrs="{'invisible': [('country_code', '!=', 'EG')]}"/>
```

Comment:
could we put it after Tax Scope or something?  We put those settings mostly on the right.